src/modules/Builder/features/BuilderAppletSettings/BuilderAppletSettings.utils.tsx

```diff
@@ -61,15 +60,15 @@ export const getSettings = ({
     {
       label: 'appletContent',
       items: [
-        {
-          icon: <Svg id="schema" />,
-          label: 'downloadSchema',
-          component: <DownloadSchemaSetting />,
-          param: SettingParam.DownloadSchema,
-          disabled: isNewApplet,
-          tooltip,
-          'data-testid': `${dataTestid}-download-schema`,
-        },
+        // {
```

Comment:
pls add short description why this field temporarily commented

Reply:
done